### PR TITLE
Formatted NFT identifier in api response

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Formatted the NFT identifier in `/address/:address/esdt` endpoint. 
Now, if a token is regular esdt, the key for it would follow the syntax `TICKER-3e3e3e` and if it the token is a NFT, then it would follow the syntax `TICKER-3e3e3e-02` where `02` is the hex encoded nonce.

Before this PR, the token identifier for NFTs would have resulted in non readable characters

### CODE REVIEW

**Review by sasurobert — APPROVED** [review]
Maybe a unit test? :)

**Review by sasurobert — APPROVED** [review]

File: cmd/node/config/api.toml [hunk 1]
```diff
@@ -49,10 +49,7 @@
         { Name = "/:address/esdt/:tokenIdentifier", Open = true },
 
         # /address/:address/nft/:tokenIdentifier/nonce/:nonce will return data of an nft esdt token for a given account, tokenID and nonce
```
Comment (miiu96) [inline cmd/node/config/api.toml:51]: `data of a nft` instead `data of an nft`

File: node/node.go [hunk 2]
```diff
@@ -60,6 +60,8 @@ import (
 // SendTransactionsPipe is the pipe used for sending new transactions
 const SendTransactionsPipe = "send transactions pipe"
 
+const esdtTickerNumChars = 6
```
Comment (miiu96) [inline node/node.go:63]: group these constants 
const(
 ..
 ..
)

File: node/node_test.go [hunk 3]
```diff
@@ -390,6 +390,73 @@ func TestNode_GetAllESDTTokens(t *testing.T) {
 	assert.Equal(t, esdtData, value[esdtToken])
 }
 
+func TestNode_GetAllESDTTokensShouldReturnEsdtAndFormattedNft(t *testing.T) {
+	acc, _ := state.NewUserAccount([]byte("newaddress"))
```
Comment (miiu96) [inline node/node_test.go:394]: t.Parallel ?

File: cmd/node/config/api.toml (hunk 1 above)
Comment (bogdan-rosianu) [reply cmd/node/config/api.toml:51]: done

File: node/node.go (hunk 2 above)
Comment (bogdan-rosianu) [reply node/node.go:63]: done

File: node/node_test.go (hunk 3 above)
Comment (bogdan-rosianu) [reply node/node_test.go:394]: no test in this file contains `t.Parallel()` so I would leave it as it is

**Review by sasurobert — DISMISSED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**